pkg/providers/tinkerbell/config/template-cp.yaml

```diff
@@ -130,7 +133,10 @@ spec:
 {{- if .kubeletExtraArgs }}
 {{ .kubeletExtraArgs.ToYaml | indent 10 }}
 {{- end }}
-{{- if .controlPlaneTaints }}
+{{- if not .workerNodeGroupConfigurations }}
+        taints: []
+{{- end }}
+{{- if and .workerNodeGroupConfigurations .controlPlaneTaints }}
```

Comment:
Do we explicitly prevent users from specifying control plane taints but no worker node groups? As is, the taints will be ignored. Have we made a not to document this somewhere?

Reply:
Right, I have made another CR to prevent users from specifying control plane taints: https://github.com/aws/eks-anywhere/blob/main/pkg/api/v1alpha1/cluster.go#L442-L444